### PR TITLE
Bug 2022399: Fix troubleshoot link for MON_DISK_LOW

### PR DESCRIPTION
**Pull request by rexagod**
Based on https://bugzilla.redhat.com/show_bug.cgi?id=2022399#c5

### CODE REVIEW

**Comment by afreen23** [comment]
/hold

Need some discussion

**Comment by afreen23** [comment]
/hold cancel

@rexagod can you update PR ?

**Comment by rexagod** [comment]
@afreen23 I'm currently awaiting Erin's response on https://bugzilla.redhat.com/show_bug.cgi?id=2022399#c12

Please let me know if I should make the changes anyway and I'll send a patch.

Thanks.

**Comment by rexagod** [comment]
@afreen23 Should I wait for this to be merged to do `/cherry-pick release-4.9`?

**Review by afreen23 — APPROVED** [review]
/lgtm

**Comment by rexagod** [comment]
Removed `?lb_target=preview` from the preview link, as mentioned in the "Re: 4.9 ODF DR preview docs preview links" mail.

> Check the preview URL in the address bar for the ?lb_target=preview before the "#" symbol in the address bar. If this is missing, you need to manually add it to the URL in the address bar and hit enter key to refresh. You should now be able to view the content.

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelisted-health-checks.ts [hunk 1]
```diff
@@ -1,6 +1,4 @@
-// TODO(@rexagod): Add KCS/doc links once available.
-
 export const whitelistedHealthChecksRef = {
   MON_DISK_LOW:
-    'https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/troubleshooting_openshift_data_foundation/troubleshooting-alerts-and-errors-in-openshift-data-foundation_rhocs#resolving-cluster-health-issues_rhodf',
+    'https://dxp-docs.ext.us-west.aws.prod.paas.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html-single/troubleshooting_openshift_data_foundation/index#resolving-cluster-health-issues_rhodf',
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelisted-health-checks.ts:3]: Why are we not doing access.redhat.com ?
The docs are live under that only ?

Comment (rexagod) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelisted-health-checks.ts:3]: Olive told me that the links based on `access.redhat.com` are broken currently. The redirection (for `access.redhat.com`) does not seem to be working at the moment, and the customer portal has been moved to this newer domain.

Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelisted-health-checks.ts:3]: we need to make sure that what we add here will be live after release of odf 4.9.
If its broken now then its okay, the url should be fixed for future.

**Review by afreen23 — APPROVED** [review]
/lgtm

**Pull request merged**